### PR TITLE
luametatex: fix build on older systems with gcc

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Fix

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by pmetzger** [comment]
You should be able to fix the lint.

Am I right that your patch unconditionally removes debug symbols?

**Comment by barracuda156** [comment]
@pmetzger I can make the patch conditional, however you advise. I have no idea where that target succeeds or fails and have no way to check it.

Lint can be done, sure.

File: tex/luametatex/files/patch-fix-gcc-build.diff [hunk 1]
```diff
@@ -0,0 +1,9 @@
+--- cmake/luametatex.cmake	2024-03-11 16:34:51.000000000 +0800
++++ cmake/luametatex.cmake	2024-06-22 04:18:37.000000000 +0800
+@@ -86,5 +86,5 @@
+     # -g -S -d : remove all debugging symbols & sections
+     # -x       : remove all non-global symbols
+     # -X       : remove any compiler-generated symbols
+-    add_custom_command(TARGET luametatex POST_BUILD COMMAND ${CMAKE_STRIP} -g -S -d -x luametatex${CMAKE_EXECUTABLE_SUFFIX})
++    # add_custom_command(TARGET luametatex POST_BUILD COMMAND ${CMAKE_STRIP} luametatex${CMAKE_EXECUTABLE_SUFFIX})
```
Comment (pmetzger) [inline tex/luametatex/files/patch-fix-gcc-build.diff:8]: If you're commenting this out, why change the text of the line?

Comment (barracuda156) [reply tex/luametatex/files/patch-fix-gcc-build.diff:8]: @pmetzger Fair enough, that is unneeded. (I tried dropping args sequentially and it still kept failing.)

Comment (barracuda156) [reply tex/luametatex/files/patch-fix-gcc-build.diff]: @pmetzger Done

**Comment by barracuda156** [comment]
@pmetzger Second commit add homepage and fixes alignments.

**Pull request merged**